_posts/2021-09-21-being-right-is-just-the-beginning.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Being Right Is Just the Beginning"
+date: 2021-09-21
+image: "being-right.jpeg"
+category: Security
+tags: infosec, conference
+author: leigh_hall
+summary: "As an industry we’re obsessed with being smart. And that’s ok - good even, in the right context. But to get what we want we need to stop being the smartest person in the room and start being the most helpful in the room"
+---
+
+On the 8th September I was very grateful to have the opportunity to speak at my local security/hacker conference DC151 in Leeds.
+
+In the talk I lamented my lack of drive in pursuing purely technical content anymore (there's plenty of that in my [old blog](https://medium.com/@leighhall)) instead reflecting my current career arc and giving a talk on "boring management stuff", as I prefaced.
+
+Of course, I don't think it's boring or particularly constrained to management stuff, but that was the first layer of my subterfuge and was reflected back in the organiser's comment that DC151 hadn't seen a talk like this before.
+
+Nice.
+
+So anyway, this is a brief outline of my talk. Well, of my point, really, which is actually pretty simple, all told.
+
+The most succint summary of what I was saying has since come from the co-organiser of DC151, Glenn:
+
+["We need to stop being the smartest in the room and start being the most helpful in the room."](https://twitter.com/GlennPegden/status/1437063162432524290?s=20)
```

Comment:
We do have a precedent for using embedded tweets elsewhere: https://publish.twitter.com/?hideConversation=on&query=https%3A%2F%2Ftwitter.com%2FGlennPegden%2Fstatus%2F1437063162432524290&widget=Tweet